src/client/components/Editor/Editor.js

```diff
@@ -145,8 +145,9 @@ class Editor extends React.Component {
   }
 
   setBodyAndRender(body) {
+    const parsedBody = body.replace(/^\s+</gm, '<');
```

Comment:
Do we need it there? `bodyHTML` is passed to `Body` component that already strips that whitespace.

Reply:
You're right. I must have left the old regex in the body when I was testing the editor preview. Didn't notice `Write` called `getHtml` before passing the body to the Editor.